[docs] Refactor 'Intercept HTTP Requests' (#4950)
DevExpress/testcafe: docs/articles/documentation/templates/intercept-http-requests/request-filter.md

    .respond(/*...*/);
```

### Filter with a Predicate

> You can get more request parameters using a predicate function and use them to determine whether to handle the request.

Reviewer: ```suggestion
You can get more request parameters and determine whether to handle the request with the predicate functions.
```
